app/src/main/java/com/nearsoft/referralsapp/JobListing.java

```diff
@@ -0,0 +1,82 @@
+package com.nearsoft.referralsapp;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.widget.TextView;
+
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.StringRequest;
+import com.android.volley.toolbox.Volley;
+
+import java.util.ArrayList;
+
+public class JobListing extends AppCompatActivity {
+
+    private RequestQueue mRequestQueue;
+    private static final String TAG = "TAG";
+    private RecyclerView mRecyclerView;
+    private LinearLayoutManager mLayoutManager;
+    private RecyclerView.Adapter mAdapter;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_job_listing);
+        mRecyclerView = (RecyclerView) findViewById(R.id.recycler_view);
+
+        // Instantiate the RequestQueue.
```

Comment:
Suggestion: don't use comments to explain your code... remember "Code that need to be documented is code that smells" ... create auto-documented code